lib/l10n/app_id.arb

```diff
@@ -222,12 +222,12 @@
   "moderate": "Sedang",
   "deselect_all": "Batalkan Semua",
   "select_all": "Pilih Semua",
-  "are_you_sure": "Anda yakin?",
-  "generating_playlist": "Menghasilkan daftar putar khusus Anda...",
+  "are_you_sure": "Kamu yakin?",
+  "generating_playlist": "Menghasilkan daftar putar khususmu...",
```

Comment:
` "generating_playlist": "Menghasilkan daftar putar khususmu...",`
Would be better
` "generating_playlist": "Membuat daftar putar khusus kamu...",`